install/fedora-installation-guide.md

```diff
@@ -6,48 +6,42 @@ Kata Containers is available for Fedora\* versions **26** and **27**.
 This step is only required in case Docker is not installed on the system.
 1. Install the latest version of Docker with the following commands:
 
-```bash
-$ sudo dnf -y install dnf-plugins-core
-$ sudo dnf config-manager --add-repo https://download.docker.com/linux/fedora/docker-ce.repo
-$ sudo dnf makecache
-$ sudo dnf -y install docker-ce
-```
+       bash
```

Comment:
As mentioned, our CI checks these code blocks so once the PR lands to allow indented blocks this will not work as desired. The fix is to put back the backticks (here and elsewhere in the doc):

```

    ```bash

```